### ISSUE TITLE
grad_input.ipynb

### ISSUE THREAD

**Issue by jeffmylife** [kipoi/kipoi#304]
The [documentation](http://kipoi.org/docs/using/02_Postprocessing/#gradient-calculation) mentions a gradient input notebook but I cannot find it. Is it not implemented yet? 

> Python / R API
All of the arguments mentioned above are available within the grad_input method of Kipoi models that implement GradientMixin. Namely those are KerasModel, PyTorchModel, and TensorflowModel. For an example please refer to the grad_input.ipynb.
The returned output from the grad_input matches the structre of the inputs entry of the data batch generated by the model dataloader.

**Comment by krrome** [comment]
That's a bit of a backlog. It's all implemented and the notebook is
written, I just have to give it a final polish and then I'll add it. I'm
sorry that the docs already mention it.

**Comment by jeffmylife** [comment]
All good, thanks.
